.jenkins/github/pre-merge.jenkinsfile

```diff
@@ -268,7 +268,7 @@ def examples_verify() {
         dir('cumulocity-examples') {
           stage('examples verify') {
             try {
-              sh ".jenkins/scripts/mvn.sh install"
+              sh ".jenkins/scripts/mvn.sh install -Dskip.agent.package=true"
```

Comment:
Out of curiosity why did you add this?

Reply:
Because at this stage we only care about compilation and running tests in examples. We don't need need to create packages, as we don't use them anywhere. So it saves some tome and resources and will no longer fail if there are i.e. issues with Docker registry that I got one some of the builds. Also, microservice package plugin is tested well enough by building agents packages, that are used in tests later.